src/Uno.UI.Wasm/ts/Windows/UI/Xaml/Application.ts

```diff
@@ -1,14 +1,31 @@
 ﻿namespace Windows.UI.Xaml {
 
-    export class Application {
-        public static getDefaultSystemTheme(): string {
-            if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
-                return ApplicationTheme.Dark;
-            }
-            if (window.matchMedia("(prefers-color-scheme: light)").matches) {
-                return ApplicationTheme.Light;
-            }			
+	export class Application {
+		private static dispatchThemeChange: () => number;
+
+		public static getDefaultSystemTheme(): string {
+			if (window.matchMedia) {
+				if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
+					return ApplicationTheme.Dark;
+				}
+				if (window.matchMedia("(prefers-color-scheme: light)").matches) {
+					return ApplicationTheme.Light;
+				}
+			}
 			return null;
         }
+
+		public static observeSystemTheme() {
+			if (!this.dispatchThemeChange) {
+				this.dispatchThemeChange = (<any>Module).mono_bind_static_method("[Uno] Windows.UI.Xaml.Application:DispatchSystemThemeChange");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)